policy/release/labels.rego

```diff
@@ -146,15 +190,90 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
-image_labels contains label if {
-	some name, value in input.image.config.Labels
-	count(value) > 0
-	label := {"name": name, "value": value}
+# METADATA
+# title: Inaccessible parent image manifest
+# description: >-
+#   The parent image manfest is not accessible.
+# custom:
+#   short_name: inaccessible_parent_manifest
+#   failure_msg: Manifest of the image %q, parent of image %q is inaccessible
+#   solution: >-
+#     Check the provided authentication configuration and the credentials
+#     within it.
+#   collections:
+#   - redhat
+#
+deny contains result if {
+	manifest := ec.oci.image_manifest(input.image.ref)
+
+	name := manifest.annotations["org.opencontainers.image.base.name"]
+	digest := manifest.annotations["org.opencontainers.image.base.digest"]
```

Comment:
If these annotations are not present, do we expect we already produced a "required_labels" violation? That would make sense to me, but could consider a comment to mention that.

Reply:
`required_labels` should be independent of the parent labels correlation checks, if these are not present that means the image has no (recorded) parent